### ISSUE TITLE
Site specific network config

### ISSUE THREAD

**Issue by mattsm** [lgirdk/boardfarm#115]
For LAN / WAN to reach the internet- we should apply proxy settings from the global boardfarm config.

**Comment by Parumugam** [comment]
Matthew, the WAN container will also have Internet access right? The issue was with LAN Container and today, as per Lynn, this seems to be issue with "First Install" which was blocking the internet access for the LAN Container.

Do we still need to apply proxy settings to get Internet access?

**Comment by mattsm** [comment]
Yes possibly, this is a place holder for finding and implementing solutions to these issues.
